spec/requests/doj_dashboard_spec.rb

```diff
@@ -66,17 +65,21 @@
         GlobalState.open_submission_window!
```

Comment:
No, it looks like that won't work:

```
 RuntimeError:
       let declaration `elvis` accessed in a `before(:context)` hook at:
         /bridge/spec/requests/doj_dashboard_spec.rb:76:in `block (4 levels) in <top (required)>'

       `let` and `subject` declarations are not intended to be called
       in a `before(:context)` hook, as they exist to define state that
       is reset between each example, while `before(:context)` exists to
       define state that is shared across examples in an example group
```

RSpec is picky about what can go inside a `before :all` hook, which makes sense given how much state is reset between tests.